update add.md --to-remote description (#4035)
iterative/dvc.org: content/docs/command-reference/add.md

  [example](#example-transfer-to-an-external-cache)).

> - `--to-remote` - add a target that's outside the project, but neither cache it

Reviewer: ```suggestion
- `--to-remote` - add a target that's outside the project, neither move it into
```

>   nor place it in the workspace yet.

Reviewer: ```suggestion
  the workspace, nor cache it.
```
